lib/HLSL/HLOperationLower.cpp

```diff
@@ -9098,7 +9096,7 @@ void TranslateHLSubscript(CallInst *CI, HLSubscriptOpcode opcode,
     Type *HandleTy = hlslOP->GetHandleType();
     if (ptr->getType() == hlslOP->GetNodeRecordHandleType()) {
       DXASSERT(false, "Shouldn't get here, NodeRecord subscripts should have "
-                      "generated ExtractRecordStructFromArray intrinsic");
+                      "generated DefaultSubscript intrinsic");
```

Comment:
This should say something different, like:
"NodeRecord subscripts should have been lowered in LowerRecordAccessToGetNodeRecordPtr"